server/src/curfu/__init__.py

```diff
@@ -1,9 +1,15 @@
 """Fusion curation interface."""
 import logging
+from importlib.metadata import PackageNotFoundError, version
 from os import environ
 from pathlib import Path
 
-from .version import __version__
+try:
+    __version__ = version("{{ cookiecutter.project_slug }}")
```

Comment:
```suggestion
    __version__ = version("curfu")
```

Reply:
noooooo

Reply:
This is how I felt with `@main` reference 😭